pkg/command/install/uninstall.go

```diff
@@ -70,7 +72,7 @@ func (o *UninstallOptions) Complete(ctx *context.RWContext, cmd *cobra.Command,
 // TODO: need record which versions installed.
 // TODO: need to ensure no instances running before uninstall. Issue: https://github.com/risingwavelabs/risingwave-operator/issues/184
 func (o *UninstallOptions) Run(ctx *context.RWContext, cmd *cobra.Command, args []string) error {
-	yamlFile, err := Download(RisingWaveUrl, TemDir+"/operator")
+	yamlFile, err := Download(fmt.Sprintf(RisingWaveUrlTemplate, minimumVersion(MinimumVersion)), TemDir+"/operator")
```

Comment:
I will optimize this with the issue(https://github.com/risingwavelabs/risingwave-operator/issues/184)